sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java

```diff
@@ -159,36 +165,29 @@ void setFullscreen(boolean fullscreen) {
     if (null == windowManager) {
         return;
     }
-    boolean changed = this.fullscreen != fullscreen;
-    ViewGroup.LayoutParams layoutParams = getLayoutParams();
-
 
-    if (changed) {
-      if (fullscreen) {
-        //Store the source size of the layout
-        sourceWidth = layoutParams.width;
```

Comment:
do you no longer need the sourceWidth and height? Remember we needed it to know the original width and height before going into fullscreen mode.